docs/contribute.md

```diff
@@ -1,328 +1,141 @@
 
-# Read the content of the JSON file and index it in Elasticsearch
-$ bin/ralph read -b swift foo.json | \
-    bin/ralph write -b es
+### 4. Code of conduct 
 
-# Check that we have properly indexed the JSON file in Elasticsearch
-$ bin/ralph read -b es
-{"id": 1, "foo": "bar"}
-```
+Remember to maintain a positive and friendly atmosphere in discussions. 
+Use discussions as a space for productive interactions and community building.
 
-## Ralph's tray
+## Pull Requests (PR)
 
-Ralph is distributed along with its tray (a deployable package for Kubernetes
-clusters using [Arnold](https://github.com/openfun/arnold)). If you intend to
-work on this tray, please refer to Arnold's documentation first.
+Contributing to [Your Open Source Project Name] through pull requests is a powerful way to enhance the project. If you want to make changes or add new features, please follow these steps to submit a PR:
 
-Dependencies:
+### 1. Fork the Repository
 
-- [Kubectl](https://kubernetes.io/docs/tasks/tools/) (>`v.1.23.5`):
-  This CLI is used to communicate with the running Kubernetes instance you
-  will use.
-- [k3d](https://k3d.io/) (>`v.5.0.0`): This tool is used to set up
-  and run a lightweight Kubernetes cluster, in order to have a local
-  environment (it is required to complete quickstart instructions below to
-  avoid depending on an existing Kubernetes cluster).
-- [curl](https://curl.se/) is required by Arnold's CLI.
-- [gnupg](https://gnupg.org/) to encrypt Ansible vaults passwords and
-  collaborate with your team.
+Begin by forking Ralph project's repository. 
+This creates a copy of the project in your GitHub account that you can modify.
 
-### Create a local `k3d` cluster
+### 2. Clone Your Fork
 
-To create (or run) a local kubernetes cluster, we use `k3d`. The cluster's
-bootstrapping should be run _via_:
+Clone your forked repository to your local machine and change the directory to the project folder using the following commands (replace `[your_fork]` with your GitHub username):
 
 ```bash
-$ make k3d-cluster
+git clone https://github.com/<your_fork>/ralph.git
+cd ralph
 ```
 
-> Running a k3d-cluster locally supposes that the 80 and 443 ports of your
-> machine are available, so that the ingresses created for your project
-> responds properly. If one or both ports are already used by another service
-> running on your machine, the `make k3d-cluster` command may fail.
+### 3. Create a New Branch
 
-You can check that your cluster is running using the `k3d cluster` command:
+Create a new branch for your changes, ideally with a descriptive name:
 
 ```bash
-$ k3d cluster list
-NAME     SERVERS   AGENTS   LOADBALANCER
-ralph    1/1       0/0      true
+git checkout -b your-new-feature
 ```
 
-As you can see, we are running a single node cluster called `ralph`.
+### 4. Make Your Changes 
 
-### Bootstrap an Arnold project
+Implement the changes or additions to the code, ensuring it follows coding standards and is well-documented.
```

Comment:
This is mentionned below with "When committing your changes, please adhere to [OpenFUN code style and practices]". 
```suggestion
```